### PR TITLE
Add option to define the route on a `isDetailLink` field

### PR DESCRIPTION
**Pull request by Xennis**
By default the `isDetailLink` field is linked to the edition view (or to the show view, if it is linked to a read only entity). I think there a cases, where you don't want to jump form the list view to the edition view of a specific item. That's why I added the following:

`detailLinkRoute(string)`
Define the route for a link in the list view, i.e. `isDetailLink` is true. The default is `edit`, hence the link points to the edition view. The other option is `show` to point to the show view.

Let me know, if I should change or improve something.

### CODE REVIEW

File: src/javascripts/ng-admin/Main/component/service/config/Field.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ define(function (require) {
         format: 'yyyy-MM-dd',
         template: defaultValueTemplate,
         isDetailLink: false,
+        detailLinkRoute: '',
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Main/component/service/config/Field.js]: just set the default value to 'edit' here and you won't need the addition line 50.

**Comment by fzaninotto** [comment]
Good addition. I hope the little fix I asked will retrigger the e2e tests correctly, because SauceLabs seems to have failed for no reason on your last commit.

**Comment by Xennis** [comment]
I fixed it, but Travis CI still failed. Do you have an idea why? It's strange, because I only changed the readme file in the third commit, where it failed for the first time.

**Comment by Xennis** [comment]
The first error, which I found in the details of Travis CI (the numbers in the beginning are the line numbers):
```
750 Selenium standalone server started at http://127.0.0.2:55489/wd/hub
751  [launcher] Error: UnknownError: chrome not reachable
```

**Comment by fzaninotto** [comment]
I really don't understand why it fails on Travis. I'll merge it.

**Pull request merged**